snappl/psf.py

```diff
@@ -337,6 +341,29 @@ def write( self, filepath ):
         # TODO : check overwriting etc.
         yaml.dump( out, open( filepath, 'w' ) )
 
+class A25ePSF( YamlSerialized_OversampledImagePSF ):
+
+    def __init__( self, band, sca, x, y, *args, **kwargs ):
+
+        super().__init__( *args, **kwargs )
+        
+        cfg = Config.get()
+        basepath = pathlib.Path( cfg.value( 'photometry.snappl.A25ePSF_path' ) )
+
+        grid_centers = np.linspace(0.5 * 511, 4088 - 0.5 * 511, 8)
```

Comment:
Where do these magic numbers come from?

255.5, 4088-255.5 ?

I suggest using variables to document it
```
nx=4088
small_nx=256
offset = 0.5?
grid_centers = ...
```